packages/apps/reputation-oracle/server/src/modules/payout/payout.dto.ts

```diff
@@ -0,0 +1,11 @@
+export class PayoutsResultDto {
+  /**
+   * URL to the stored results.
+   */
+  url: string;
+
+  /**
+   * Boolean indicating if the checks passed.
+   */
+  checkPassed: boolean;
```

Comment:
what is checkPassed for?

Reply:
If checkPassed = false we will decrease reputation